src/components/Sidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { usePostsContext } from '../hooks/usePostsContext';
+import { PostDetails } from './Posts/PostDetails';
+
+export const Sidebar: FC = () => {
+  const { selectedPost } = usePostsContext();
+
+  return (
+    <div
+      data-cy="Sidebar"
+      className={classNames(
+        'tile',
+        'is-parent',
+        'is-8-desktop',
+        'Sidebar',
+        {
+          'Sidebar--open': selectedPost,
+        },
+      )}
+    >
+
+      {selectedPost && (
+        <div className="tile is-child box is-success">
```

Comment:
PostDetails component is used without providing required props. If this component requires any props, you should pass them.